src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-gmplot.py

```diff
@@ -0,0 +1,14 @@
+# ------------------------------------------------------------------
+# Copyright (c) 2005-2020 PyInstaller Development Team.
+#
+# This file is distributed under the terms of the Apache License 2.0
+#
+# The full license is available in LICENSE.APL.txt, distributed with
+# this software.
+#
+# SPDX-License-Identifier: Apache-2.0
+# ------------------------------------------------------------------

```

Comment:
OK, I've just been corrected by Legorooj. I was wrong about the licence. Apache is for runtime hooks, GPL if for regular ones like this one. Apologies - could you change it back.